charts/nexus-iq/README.md

```diff
@@ -99,12 +99,28 @@ The command removes all the Kubernetes components associated with the chart and
 | `deployment.preStart.command`              | Command to run before starting the IQ Server container                                                              | `nil`                                                       |
 | `deployment.postStart.command`             | Command to run after starting the IQ Server container                                                               | `nil`                                                       |
 | `deployment.terminationGracePeriodSeconds` | Update termination grace period (in seconds)                                                                        | 120s                                                        |
-| `persistence.storageClass`                 | The provisioner class                                                                                               | `-` (disables dynamic provisioning                          |
+| `persistence.storageClass`          | The provisioner class   | `-` (disables dynamic provisioning       |
 | `persistence.storageSize`                  | The amount of drive space to allocate                                                                               | `1Gi`                                                       |
 | `persistence.accessMode`                   | Default access mode                                                                                                 | `ReadWriteOnce`                                             |
 | `persistence.existingClaim`                | Pre-created PVC name for Data Volume                                                                                | `nil`                                                       |
 | `persistence.existingLogClaim`             | Pre-created PVC name for Log Volume                                                                                 | `nil`                                                       |
-| `persistence.volumeConfiguration`          | A YAML block to configure the persistent volume type. Defaults to `hostPath` which should not be used in production | `hostPath`                                                  |
+| `persistence.pdName` **DEPRECATED** | Moved to  `persistence.gcePersistentDisk.pdName` | NA |
+| `persistence.fsType` **DEPRECATED** | Moved to  `persistence.gcePersistentDisk.fsType` | NA |
+| `persistence.pvName` | The name for the persistentVolume being created to hold IQ Data | `nil` |
+| `persistence.logpvName` | The name for the persistentVolume being created to hold IQ Logs | `nil` |
```

Comment:
would all the names for the log volumes be more consistent as more recognizable lowerCamelCase: logPVName, logPDName, logFSType, logVolumeID, etc? I'm not sure I feel strongly about it, but these `log[a-z].*` names stood out to me.

Reply:
Oh! I was following the existing paradigm. I totally agree with you on this!